.changeset/deep-mangos-stand.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"@fluid-private/test-end-to-end-tests": minor
+---
+---
+"section": feature
+---
+
+Shorter ids for Datastores and DDSes
```

Comment:
```suggestion
Shorter IDs for Datastores and DDSes
```

Is Datastore the name of the code entity or is it DataStore, etc?

Reply:
Fixed. It's DataStore.